backend/posttest-script.sh

```
@@ -0,0 +1,22 @@
+
+# remove container
+if [[ "$(docker ps -a -q -f name=test-postgres)" ]]; then
+  docker rm -f test-postgres
+fi
+
+# reset NODE_ENV to default
+if [[ "$OSTYPE" == "msys" ]]; then
+  # For Windows CMD: set "NODE_ENV=development"
+  # For PowerShell: $env:NODE_ENV="development"
+  set "NODE_ENV=development"
+  $env:NODE_ENV="development"
```

Shellcheck (0.10.0)

[error] This is interpreted as a command name containing '='. Bad assignment or comparison?

(SC2276)